lib/chef-dk/skeletons/code_generator/recipes/build_cookbook.rb

```diff
@@ -87,8 +87,7 @@
 end
 
 file "#{build_cookbook_dir}/test/fixtures/cookbooks/test/metadata.rb" do
-  content %(name 'test'
-version '0.1.0')
+  content %(name 'test'\nversion '0.1.0'\n)
```

Comment:
this should really be a HEREDOC, or else it should just use double quotes around the string.